src/main/java/com/google/devtools/build/lib/analysis/starlark/StarlarkCustomCommandLine.java

```diff
@@ -169,13 +178,15 @@ static final class VectorArg {
         UUID.fromString("a4e5e090-0dbd-4d41-899a-77cfbba58655");
 
     private final int features;
+    @Nullable private final UUID elementType;
```

Comment:
Call it `nestedSetElementType` to make it clear it's only relevant for the `NestedSet` case.